plugins/flutter_plugins/README.md

```diff
@@ -0,0 +1,27 @@
+# Desktop Implementation of flutter/plugins Plugins
+
+Each plugin in this directory corresponds to a
+[`flutter/plugins`](https://github.com/flutter/plugins) plugin with the
+same name, minus the `_fde` suffix.
+
+These implementations exist here only as a temporary solution to use these
+plugins while the plugin APIs on each desktop platform stabilize enough that
+they can move to an official location and be distributed as normal Flutter
+plugins.
+
+## Using These Plugins
+
+For these plugins, the Dart code comes from the official plugin, so you
+must include that in your `pubspec.yaml` as well. For instance, to use
+url\_launcher on desktop, you would include:
```

Comment:
url_launcher*

Reply:
Resolved offline; this was referring to the `\` which is intentional escaping.